uint8[] image
std_msgs/Header header
aarapsi_robot_pack/ImageStructure data
